app/src/main/java/org/oppia/app/player/content/ContentListFragmentPresenter.kt

```diff
@@ -0,0 +1,57 @@
+package org.oppia.app.player.content
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import org.oppia.app.databinding.ContentListFragmentBinding
+import org.oppia.app.viewmodel.ViewModelProvider
+import org.oppia.data.backends.gae.model.GaeSubtitledHtml
+import org.oppia.domain.exploration.ExplorationProgressController
+import org.oppia.util.logging.Logger
+import javax.inject.Inject
+
+/** Presenter for [ContentListFragment]. */
+class ContentListFragmentPresenter @Inject constructor(
+  private val fragment: Fragment,
+  private val viewModelProvider: ViewModelProvider<ContentViewModel>,
+  private val logger: Logger
+) {
+
+  private val entity_type: String = "exploration"
+
+  var contentCardAdapter: ContentCardAdapter? = null
```

Comment:
You can make this lateinit and remove this null support.

Reply:
changed to lateinit.